src/components/OrganizationStepper.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+
+import { makeStyles } from '@material-ui/core/styles';
+import Stepper from '@material-ui/core/Stepper';
+import Step from '@material-ui/core/Step';
+import StepLabel from '@material-ui/core/StepLabel';
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+import ScholarshipForm from './ScholarshipForm';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: '100%',
+  },
+  button: {
+    marginRight: theme.spacing(1),
+  },
+  instructions: {
+    marginTop: theme.spacing(1),
+    marginBottom: theme.spacing(1),
+  },
+}));
+
+function getSteps() {
+  return ['Step One', 'Step Two', 'Step Three'];
+}
+
+function getStepContent(step) {
+  switch (step) {
+    case 0:
+      return 'Provide us with the scholarship details in Step Two and we will ensure to add the scholarship to our database. We appreciate your contribution!';
+    case 1:
+      return 'Please Provide Scholarship Details Below :]';
+    case 2:
+      return 'To complete the submission of the scholarship click on the Submit button. We would love to have you on our team as a moderator! If you are intrested once you submit click on the button Join Team to learn more.';
+    default:
+      return 'Unknown step';
+  }
```

Comment:
I don't think this is how we'll ask organizations to provide scholarship details.

Reply:
This file no longer exist